WordPress/src/main/res/layout/page_virtual_homepage_item.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
💡 Would it make sense to have the `ConstraintLayout` directly in the root of this file and remove the wrapping `LinearLayout`?

Reply:
I just copied over the layout for `page_list_item` for this one, so to keep things consistent I left this part as is, having the divider be a sibling of the `ConstraintLayout` instead of a child, therefore needing the `LinearLayout` as a parent.